### ISSUE TITLE
Slideshow with older Browsers

### ISSUE THREAD

**Issue by fabian1512** [immichFrame/ImmichFrame#181]
### Description

Unfortunately, it is currently not possible to display this fantastic slideshow on older browsers. 
I would be delighted if the compatible browsers were extended to older versions.
I would be happy to do without optical functions if they would no longer work due to compatibility.

### Use Case

I think older devices in particular can still be given a reason for existence with the function of a digital picture frame.
In my case, I would like to breathe new life into an iPad 4 with iOS 10.

### Pre-Submission Checklist

- [X] This is a feature request and not a bug report

- [X] I have checked for related issues and checked the documentation

**Comment by JW-CH** [comment]
Could you elaborate on 'older browsers'? What are you running on? How can you tell it is not running in 'older browsers'?

**Comment by JW-CH** [comment]
Could you share some version details of the browser? A safari version would be helpful.

**Comment by fabian1512** [comment]
Sorry, I cannot determine the Safari-Version. IOS-Version is 10.3.4 
A screenshot would not help, as it stays white. On my Mac, the slideshow on the dedicated ip:port  is working properly.
Nevertheless, thank you very much for accepting my request

**Comment by 3rob3** [comment]
We are seeing same issue with older Android versions (and the old Chrome versions they come with). Are you able to install Chrome (or other browser) on your iPad? Maybe that would be new enough? Probably not as they all really just shell the webkit browser.

**Comment by fabian1512** [comment]
Unfortunately, I cannot install older browsers. 
Apple refuses to register current Apple IDs on old devices. And the newly created ID cannot load older versions. Even a jailbreak didn't help.
However, as far as I have read and understood, all browsers use the iOS engine.

**Comment by 3rob3** [comment]
We tried to work around this same issue with older Chromium and webviews with no luck. Closing this, not much we can do about it at this point.

**Comment by ZeroZeroOne-dev** [comment]
Ah this is unfortunate, I have set up immichFrame only to find out it doesn't work on my old second gen iPad I wanted to use as a picture frame😟

**Comment by 3rob3** [comment]
> Ah this is unfortunate, I have set up immichFrame only to find out it doesn't work on my old second gen iPad I wanted to use as a picture frame😟

Are you able to install another browser?

**Comment by ZeroZeroOne-dev** [comment]
No, iOS 9.3.5 is the last supported version on this iPad. No apps can be downloaded from the app store anymore that work with it.

**Comment by 3rob3** [comment]
A lot of us are running the cheap Frameo photo frame's you can get on Amazon for ~$40. You can sideload the ImmichFrame Android app on there.

**Comment by ZeroZeroOne-dev** [comment]
Thanks, I'll check those out as an alternative!
